fdbserver/BlobWorker.actor.cpp

```diff
@@ -3803,12 +3808,12 @@ ACTOR Future<Void> doBlobGranuleFileRequest(Reference<BlobWorkerData> bwData, Bl
 							    .detail("Encrypted", encrypted);
 						}
 
-						if (g_network && g_network->isSimulated() &&
-						    isEncryptionOpSupported(EncryptOperationType::BLOB_GRANULE_ENCRYPTION) && !encrypted) {
+						if (g_network && g_network->isSimulated() && isEncryptionOpSupported(bwData->encryptMode) &&
```

Comment:
nit: how about transforming it to:

`ASSERT(!(g_network && g_network->isSimulated() && isEncryptionOpSupported(bwData->encryptMode) && !encrypted)) `

Seems like `if` block is only supporting an ASSERT?